Cargo.toml

```
@@ -22,3 +26,6 @@ members = [
 
 # Required for wgpu v0.10 feature resolution.
 resolver = "2"
+
+[workspace.dependencies]
+bevy = { git = "https://github.com/bevyengine/bevy", branch = "main", features = ["default"] }
```

Ah I just noticed we point to git here! Is there a reason we use git rather than the crates version here? If not I might change this to use crates.io in #950 like so:

```suggestion
bevy = "0.12"
```

---

Bevy moves pretty quick but tends to keep the main branch stable. This was just an attempt to keep up to date with their changes while we're in "dev" mode so we don't have churn on updates from them. If it's a problem for nix, we can definitely just target a stable version.

---

I'm thinking either way we lock the dependency to a particular commit with `Cargo.lock`, but if we use a crates.io version we can choose when we want to update for bevy breakage in a dedicated PR (e.g. we might want to run `cargo update` to pull down patches across our deps without also bumping `bevy` to the latest `main` commit and needing to address any breakage)?

---

Def makes sense to specify something stable so we can track changes to the framework in dedicates PRs rather than ad hoc. I think my preference would still be for a specific SHA on `main`, but I'm really fine with either. I haven't audited what's upcoming in `0.13`, but my concern is mostly that we're using some of the less documented APIs which they break more often. If using an unreleased version causes issues I'm happy to target the latest release. We'll obviously want to target their stable version anyway once we get closer to finished. Up to you!